doc/source/ovn_kubernetes/docs/ovn-kubernetes-basics.md

```diff
@@ -0,0 +1,7 @@
+# OVN Kubernetes Basics 
+
+A good resrouce to get started with understanding `ovn-kubernetes` is the following recording and slides, which run through the basic architecture and functionality of the system. 
```

Comment:
Just a minor typo here 'resrouce->resource'

Reply:
sorry not the best speller :)